Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN find /etc/systemd/system \
 RUN systemctl set-default multi-user.target
 CMD ["/sbin/init"]
 
-RUN apt-get update; apt-get -y upgrade
-RUN apt-get -y install monit apache2
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get -y install monit 
+RUN apt install -y apache2
```

Comment:
Actually these changes are almost equivalent to the original. So they don't improve anything. The only effect of making these changes is that the cache is invalidated and that part of the image is built again.